clouds.yaml.template

```diff
@@ -1,9 +1,24 @@
 clouds:
+{{- if eq .AuthType "none"}}
   metal3-bootstrap:
-    auth_type: none
+    auth_type: {{.AuthType}}
     baremetal_endpoint_override: {{.BootstrapIronicURL}}
     baremetal_introspection_endpoint_override: {{.BootstrapInspectorURL}}
   metal3:
-    auth_type: none
+    auth_type: {{.AuthType}}
     baremetal_endpoint_override: {{.ClusterIronicURL}}
     baremetal_introspection_endpoint_override: {{.ClusterInspectorURL}}
+{{- else if eq .AuthType "http_basic"}}
+  metal3:
```

Comment:
Why `metal3-bootstrap` has been removed?

Reply:
since I moved the `generate_template` to be at the end of `create_cluster`  the bootstrap will be gone by that time, the reason for this is that the `clouds.yaml` for `http_basic` requires the metal3 pod to be up so we can get the username/password for ironic and inspector

Reply:
Ok, so how it will affect the old procedure for debugging bootstrap VM, as documented at https://github.com/openshift-metal3/dev-scripts/#interacting-with-ironic-directly?

Reply:
I'm going to update the metal3-templater.go  adding an option to only generate the bootstrap info (so it can be called at the begin of `create_cluster` ) I think this will solve our problems. Will try to test locally and I will update the PR